app/helpers/pagination_helpers.rb

```diff
@@ -0,0 +1,19 @@
+module PaginationHelpers
+  extend ActiveSupport::Concern
+
+  private
+
+  # Override this method in your class to change the default
+  def default_page_size
+    5
+  end
+
+  def page_size
+    limit = params[:limit].to_i
+    limit.positive? ? limit : default_page_size
```

Comment:
do we want to add a max page size as well?
```ruby
limit.positive? ? [limit, maximum_page_size].min : default_page_size
```

Reply:
I think this is a good idea - just to prevent users trying to load too much data at a time I guess

Reply:
I don't need to add this logic, there is a kaminari_config file in config/initialisers that has a setting for max_per_page